java/algorithm/pom.xml

```diff
@@ -50,4 +55,68 @@
 
   <build>
   </build>
+
+  <profiles>
+    <profile>
+      <id>spotless-jdk11+</id>
```

Comment:
Do we need a `spotless-jdk8`?

Reply:
I assume we still do have some JDK 8 developers? That's why I wanted to leave it here, but if it is not a very strong case, we can definitely remove it. WDYT?

Reply:
I don't want to have two separate versions of the formatter. Only build/test needs to work on JDK8.